packages/arb-token-bridge-ui/src/components/common/Layout.tsx

```diff
@@ -36,6 +37,18 @@ export function Layout(props: LayoutProps) {
       style={{ backgroundImage: 'url(/images/space.webp)' }}
       className="background-image relative flex min-h-screen flex-col overflow-hidden bg-repeat"
     >
+      <SiteBanner>
+        The Stylus testnet is officially live, build EVM-compatible apps in Rust
```

Comment:
Did we receive this copy from product? I'd assume Stylus Testnet is the network name and should be capitalized, but may be wrong here.

Reply:
Could be interpreted either way tbh, can leave it lowercase I guess